cves/CVE-2017-14058.yml

```diff
@@ -346,7 +351,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: This is a big example of design mistake, lack of testing, and miscommunication. A denial of service occuring because of logs or reloads being spammed is something that should always be looked for as that is a well known security problem. This lasting in the system for 6 years is a big oversight that should have been noticed earlier.
```

Comment:
Very long line, should reformat to stay within the format of the rest of the file (80 chars per line)

Reply:
thank you fixed